web/src/components/ModalZeroButton.tsx

```diff
@@ -2,19 +2,33 @@ import { Heading } from "@/components/njwds-extended/Heading";
 import { Icon } from "@/components/njwds/Icon";
 import { ContextualInfoContext } from "@/contexts/contextualInfoContext";
 import { Dialog, DialogContent, DialogTitle, IconButton } from "@mui/material";
-import { ReactElement, ReactNode, useContext } from "react";
+import { ReactElement, ReactNode, useContext, useEffect, useRef } from "react";
 
 interface Props {
   isOpen: boolean;
   close: () => void;
   title: string;
   children: ReactNode;
   unpaddedChildren?: ReactNode;
-  uncloseable?: boolean;
 }
 
 export const ModalZeroButton = (props: Props): ReactElement => {
   const { contextualInfo } = useContext(ContextualInfoContext);
+
+  const dialogRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (dialogRef.current) {
+      const firstFocusableElement = dialogRef.current.querySelector<HTMLElement>(
+        'button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"])'
+      );
+      if (firstFocusableElement) {
+        firstFocusableElement.focus();
+      }
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [props.isOpen, dialogRef.current]);
```

Comment:
I'm disabling a warning here. This is because usually running on ref.current is bad, but there isn't a good way around it here because if this useEffect runs at component load time the ref is undefined as the useEffect runs before the component renders.